common/static/common/templates/components/paginated-view.underscore

```diff
@@ -1,4 +1,4 @@
 <div class="sr-is-focusable sr-<%= type %>-view" tabindex="-1"></div>
 <div class="<%= type %>-paging-header"></div>
-<div class="<%= type %>-list"></div>
+<ul class="<%= type %>-list"></ul>
```

Comment:
Yes-- Studio assets (Files and Uploads), Bookmarks (I can't remember whether or not the pagination is done yet there...).